client/p2p/discv5/udp.go

```diff
@@ -437,3 +440,46 @@ func decodePacket(buffer []byte, pkt *ingressPacket) error {
 	err = s.Decode(pkt.data)
 	return err
 }
+
+// IsDiscoveryPacket returns nil error if a packet is a DiscoveryV5 KCOIN packet
+// it's a kind of 'bad and fast' code to make dropping non-kcoin connections as fast as possible
+func IsDiscoveryPacket(buffer []byte) error {
```

Comment:
can't read this method, can you split so it's easier to  understand

Reply:
I'll try, but it's a really low-level method...